src/bun.js/test/expect.zig

```diff
@@ -732,6 +732,58 @@ pub const Expect = struct {
         return .zero;
     }
 
+    pub fn toContainKey(
+        this: *Expect,
+        globalObject: *JSC.JSGlobalObject,
+        callFrame: *JSC.CallFrame,
+    ) callconv(.C) JSC.JSValue {
+        defer this.postMatch(globalObject);
+        const thisValue = callFrame.this();
+        const arguments_ = callFrame.arguments(1);
+        const arguments = arguments_.ptr[0..arguments_.len];
+
+        if (arguments.len < 1) {
+            globalObject.throwInvalidArguments("toContainKey() takes 1 argument", .{});
+            return .zero;
+        }
+
+        incrementExpectCallCounter();
+
+        const expected = arguments[0];
+        expected.ensureStillAlive();
+        const value: JSValue = this.getValue(globalObject, thisValue, "toContainKey", "<green>expected<r>") orelse return .zero;
+
+        const not = this.flags.not;
+        const key = value.get(globalObject, expected.toString(globalObject).getZigString(globalObject).full());
```

Comment:
jest-extended uses `hasOwnProperty`. Instead of calling `.get()`, can you add a new binding to `bindings.zig/bindings.cpp` for `hasOwnProperty`. Pretty sure what JSC uses is `objectPrototypeHasOwnProperty`